python/aswfdocker/constants.py

```diff
@@ -0,0 +1,93 @@
+import enum
+from aswfdocker import versioninfo
+
+
+class ImageType(enum.Enum):
+    IMAGE = "image"
+    PACKAGE = "package"
+
+
+VERSIONS = {
+    ImageType.PACKAGE: {
+        "clang": ["1.1"],
```

Comment:
I'm don't understand 100% what this does, but it seems to specify which package goes into which image? It feels like this should belong in a file that's more like a configuration file in a centralized location, which could potentially be consumed by something else than this Python utility?

Reply:
These are all the current version numbers for the docker images, which will be used to tag the images as well when I get to implement that part :-)
I thought about moving these into a yaml file but it meant adding another python package dependency to parse yaml files... and I think I provide enough command line utilities to list the versions that keeping these in a python dictionary made sense.
For now I'm inclined to keep these here for simplicity, but happy to revisit in the future if other languages or the CI system need access as well.